src/components/Picker/BasePicker.tsx

```diff
@@ -136,6 +138,18 @@ function BasePicker<TPickerValue>(
         },
     }));
 
+    /**
+     * Windows will reuse the text color of the select for each one of the options
+     * so we need to color accordingly so it doesn't blend with the background.
+     */
+    const getItemColor = useMemo(() => {
```

Comment:
As this is useMemo (not useCallback), `itemColor` is more accurate name

Reply:
Fixed!